packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.ts

```diff
@@ -155,16 +157,61 @@ export async function injectRecaptchaFields<T>(
   auth: AuthInternal,
   request: T,
   action: RecaptchaActionName,
-  captchaResp = false
+  captchaResp = false,
+  fakeToken = false
```

Comment:
Change them to `isCaptchaResp` and `isFakeToken` which seem better names for boolean values?

Reply:
Done. Thanks!